src/scf/potential_mixing.jl

```diff
@@ -13,25 +13,32 @@ using Statistics
 #    fixed-point map          g(x)  = V + α Pf(x)
 # where the α may vary between steps.
 #
-# Finds the linear combination xₙ₊₁ = g(xₙ) + ∑ᵢ βᵢ (g(xᵢ) - g(xₙ))
-# such that |Pf(xₙ) + ∑ᵢ βᵢ (Pf(xᵢ) - Pf(xₙ))|² is minimal
+# Finds the linear combination xₙ₊₁ = g(xₙ) + ∑ᵢ βᵢ (g(xᵢ₊₁) - g(xᵢ))
+# such that |Pf(xₙ) + ∑ᵢ βᵢ (Pf(xᵢ₊₁) - Pf(xᵢ))|² is minimal
 struct AndersonAcceleration
     m::Int                  # maximal history size
```

Comment:
Add comment somewhere that ideally one chooses `m` to the maximal value fitting into main memory (trusting on `δ` and `maxcond` to take care of conditioning issues). But since it is tricky to do this reliably in an automated fashion we just pick `m=10` here by default.